app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,127 @@
+class TweetsController < ApplicationController
+
+
+	get '/tweets' do
+
+		if logged_in?
+			@tweets = Tweet.all
+
+			erb :'tweets/tweets'
+		else
+			redirect to '/login'
+		end
+	end
+
+
+	post '/tweets' do
+		if logged_in?
+			if params[:tweet] == ""
+				redirect to "/tweets/new"
+			else
+				Tweet.all.clear
+				@user = User.find_by(email: session[:user_id][:email])
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.